src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Date.txt

```diff
@@ -214,9 +215,32 @@ true
 >> Day(Date(1976,1,4) + Time(12,30,15))
 4
 
-// Excel-style date math
+// ************************* Excel-style date math ************************
+
 >> Date(2001, 2, 14) + 2
-Date(2001,2,16)
+DateTime(2001,2,16,0,0,0,0)
+
+>> Date(2001, 2, 14) - 10
+DateTime(2001,2,4,0,0,0,0)
+
+>> Date(2001, 2, 14) + 2.3
+DateTime(2001,2,16,7,12,0,0)
+
+>> Date(2001, 2, 14) - 2.3
+DateTime(2001,2,11,16,48,0,0)
+
+// Across DST
+>> Date(2022, 1, 1) + 180
+DateTime(2022,6,30,0,0,0,0)
```

Comment:
The result cannot be at midnight due to DST

Reply:
Actually, when we are adding days / months / quarters / years, this is the behavior that we get. For example, `Console.WriteLine(new DateTime(2022, 1, 1, 0, 0, 0, DateTimeKind.Local).AddDays(180).ToString("yyyy-MM-dd HH:mm:ss"));` will print out "2022-06-30 00:00:00". I added two new test files for both the US Pacific and Romance time zones just in case.